src/wallet/walletutil.h

```diff
@@ -5,6 +5,7 @@
 #ifndef BITCOIN_WALLET_WALLETUTIL_H
 #define BITCOIN_WALLET_WALLETUTIL_H
 
+#include <algorithm>
```

Comment:
Why did you add this here? If it's needed in the .cpp file, put it there...

Reply:
;was keen to see some compiler warnings, i though that was the hint from @hebasto  to add `<algorithm>` addressed and changed back to .cpp that other bug we can fix later.